File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Nuro/NuroRES1001U00ReseredDataListItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Nuro
{
    [Serializable]
    public class NuroRES1001U00ReseredDataListItemInfo
    {
        private String _receptionTime;
        private String _patientCode;
        private String _patientName1;
        private String _patientName2;
        private String _examStatus;
        private String _updDate;
        private String _pkout1001;
        private String _comingDate;
        private String _departmentCode;
        private String _receptionNo;
        private String _type;
        private String _doctorCode;
        private String _resType;
        private String _resUserName;
        private String _resInputType;
        private String _comingStatus;
        private String _newRow;
        private String _examState;
        private String _examIraiState;
        private String _resUser;
        private String _ipwonStatus;
        private String _reserComments;
        private String _reserType;
        private String _clinicName;

        public String ReceptionTime
        {
            get { return this._receptionTime; }
            set { this._receptionTime = value; }
        }

        public String PatientCode
        {
            get { return this._patientCode; }
            set { this._patientCode = value; }
        }

        public String PatientName1
        {
            get { return this._patientName1; }
            set { this._patientName1 = value; }
        }

        public String PatientName2
        {
            get { return this._patientName2; }
            set { this._patientName2 = value; }
        }

        public String ExamStatus
        {
            get { return this._examStatus; }
            set { this._examStatus = value; }
        }

        public String UpdDate
        {
            get { return this._updDate; }
            set { this._updDate = value; }
        }

        public String Pkout1001
        {
            get { return this._pkout1001; }
            set { this._pkout1001 = value; }
        }

        public String ComingDate
        {
            get { return this._comingDate; }
            set { this._comingDate = value; }
        }

        public String DepartmentCode
        {
            get { return this._departmentCode; }
            set { this._departmentCode = value; }
        }

        public String ReceptionNo
        {
            get { return this._receptionNo; }
            set { this._receptionNo = value; }
        }

        public String Type
        {
            get { return this._type; }
            set { this._type = value; }
        }

        public String DoctorCode
        {
            get { return this._doctorCode; }
            set { this._doctorCode = value; }
        }

        public String ResType
        {
            get { return this._resType; }
            set { this._resType = value; }
        }

        public String ResUserName
        {
            get { return this._resUserName; }
            set { this._resUserName = value; }
        }

        public String ResInputType
        {
            get { return this._resInputType; }
            set { this._resInputType = value; }
        }

        public String ComingStatus
        {
            get { return this._comingStatus; }
            set { this._comingStatus = value; }
        }

        public String NewRow
        {
            get { return this._newRow; }
            set { this._newRow = value; }
        }

        public String ExamState
        {
            get { return this._examState; }
            set { this._examState = value; }
        }

        public String ExamIraiState
        {
            get { return this._examIraiState; }
            set { this._examIraiState = value; }
        }

        public String ResUser
        {
            get { return this._resUser; }
            set { this._resUser = value; }
        }

        public String IpwonStatus
        {
            get { return this._ipwonStatus; }
            set { this._ipwonStatus = value; }
        }

        public String ReserComments
        {
            get { return this._reserComments; }
            set { this._reserComments = value; }
        }

        public String ReserType
        {
            get { return this._reserType; }
            set { this._reserType = value; }
        }

        public String ClinicName
        {
            get { return this._clinicName; }
            set { this._clinicName = value; }
        }

        public NuroRES1001U00ReseredDataListItemInfo() { }

        public NuroRES1001U00ReseredDataListItemInfo(String receptionTime, String patientCode, String patientName1, String patientName2, String examStatus, String updDate, String pkout1001, String comingDate, String departmentCode, String receptionNo, String type, String doctorCode, String resType, String resUserName, String resInputType, String comingStatus, String newRow, String examState, String examIraiState, String resUser, String ipwonStatus, String reserComments, String reserType, String clinicName)
        {
            this._receptionTime = receptionTime;
            this._patientCode = patientCode;
            this._patientName1 = patientName1;
            this._patientName2 = patientName2;
            this._examStatus = examStatus;
            this._updDate = updDate;
            this._pkout1001 = pkout1001;
            this._comingDate = comingDate;
            this._departmentCode = departmentCode;
            this._receptionNo = receptionNo;
            this._type = type;
            this._doctorCode = doctorCode;
            this._resType = resType;
            this._resUserName = resUserName;
            this._resInputType = resInputType;
            this._comingStatus = comingStatus;
            this._newRow = newRow;
            this._examState = examState;
            this._examIraiState = examIraiState;
            this._resUser = resUser;
            this._ipwonStatus = ipwonStatus;
            this._reserComments = reserComments;
            this._reserType = reserType;
            this._clinicName = clinicName;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/nuro/NuroRES1001U00ReseredDataListItemInfo.java
package nta.med.data.model.ihis.nuro;

import java.sql.Timestamp;

public class NuroRES1001U00ReseredDataListItemInfo {
	private String receptionTime;
	private String patientCode;
	private String patientName1;
	private String patientName2;
	private String examStatus;
	private String updDate;
	private Double pkout1001;
	private Timestamp comingDate;
	private String departmentCode;
	private Double receptionNo;
	private String type;
	private String doctorCode;
	private String resType;
	private String resUserName;
	private String resInputType;
	private String comingStatus;
	private String newRow;
	private String examState;
	private String examIraiState;
	private String resUser;
	private String ipwonStatus;
	private String reserComments;
	private String reserType;
	private String yoyangName;
	public NuroRES1001U00ReseredDataListItemInfo(String receptionTime,
			String patientCode, String patientName1, String patientName2,
			String examStatus, String updDate, Double pkout1001,
			Timestamp comingDate, String departmentCode, Double receptionNo,
			String type, String doctorCode, String resType, String resUserName,
			String resInputType, String comingStatus, String newRow,
			String examState, String examIraiState, String resUser,
			String ipwonStatus, String reserComments, String reserType, String yoyangName) {
		this.receptionTime = receptionTime;
		this.patientCode = patientCode;
		this.patientName1 = patientName1;
		this.patientName2 = patientName2;
		this.examStatus = examStatus;
		this.updDate = updDate;
		this.pkout1001 = pkout1001;
		this.comingDate = comingDate;
		this.departmentCode = departmentCode;
		this.receptionNo = receptionNo;
		this.type = type;
		this.doctorCode = doctorCode;
		this.resType = resType;
		this.resUserName = resUserName;
		this.resInputType = resInputType;
		this.comingStatus = comingStatus;
		this.newRow = newRow;
		this.examState = examState;
		this.examIraiState = examIraiState;
		this.resUser = resUser;
		this.ipwonStatus = ipwonStatus;
		this.reserComments = reserComments;
		this.reserType = reserType;
		this.yoyangName = yoyangName;
	}
	public String getReceptionTime() {
		return receptionTime;
	}
	public void setReceptionTime(String receptionTime) {
		this.receptionTime = receptionTime;
	}
	public String getPatientCode() {
		return patientCode;
	}
	public void setPatientCode(String patientCode) {
		this.patientCode = patientCode;
	}
	public String getPatientName1() {
		return patientName1;
	}
	public void setPatientName1(String patientName1) {
		this.patientName1 = patientName1;
	}
	public String getPatientName2() {
		return patientName2;
	}
	public void setPatientName2(String patientName2) {
		this.patientName2 = patientName2;
	}
	public String getExamStatus() {
		return examStatus;
	}
	public void setExamStatus(String examStatus) {
		this.examStatus = examStatus;
	}
	public String getUpdDate() {
		return updDate;
	}
	public void setUpdDate(String updDate) {
		this.updDate = updDate;
	}
	public Double getPkout1001() {
		return pkout1001;
	}
	public void setPkout1001(Double pkout1001) {
		this.pkout1001 = pkout1001;
	}
	public Timestamp getComingDate() {
		return comingDate;
	}
	public void setComingDate(Timestamp comingDate) {
		this.comingDate = comingDate;
	}
	public String getDepartmentCode() {
		return departmentCode;
	}
	public void setDepartmentCode(String departmentCode) {
		this.departmentCode = departmentCode;
	}
	public Double getReceptionNo() {
		return receptionNo;
	}
	public void setReceptionNo(Double receptionNo) {
		this.receptionNo = receptionNo;
	}
	public String getType() {
		return type;
	}
	public void setType(String type) {
		this.type = type;
	}
	public String getDoctorCode() {
		return doctorCode;
	}
	public void setDoctorCode(String doctorCode) {
		this.doctorCode = doctorCode;
	}
	public String getResType() {
		return resType;
	}
	public void setResType(String resType) {
		this.resType = resType;
	}
	public String getResUserName() {
		return resUserName;
	}
	public void setResUserName(String resUserName) {
		this.resUserName = resUserName;
	}
	public String getResInputType() {
		return resInputType;
	}
	public void setResInputType(String resInputType) {
		this.resInputType = resInputType;
	}
	public String getComingStatus() {
		return comingStatus;
	}
	public void setComingStatus(String comingStatus) {
		this.comingStatus = comingStatus;
	}
	public String getNewRow() {
		return newRow;
	}
	public void setNewRow(String newRow) {
		this.newRow = newRow;
	}
	public String getExamState() {
		return examState;
	}
	public void setExamState(String examState) {
		this.examState = examState;
	}
	public String getExamIraiState() {
		return examIraiState;
	}
	public void setExamIraiState(String examIraiState) {
		this.examIraiState = examIraiState;
	}
	public String getResUser() {
		return resUser;
	}
	public void setResUser(String resUser) {
		this.resUser = resUser;
	}
	public String getIpwonStatus() {
		return ipwonStatus;
	}
	public void setIpwonStatus(String ipwonStatus) {
		this.ipwonStatus = ipwonStatus;
	}
	public String getReserComments() {
		return reserComments;
	}
	public void setReserComments(String reserComments) {
		this.reserComments = reserComments;
	}
	public String getReserType() {
		return reserType;
	}
	public void setReserType(String reserType) {
		this.reserType = reserType;
	}
	public String getYoyangName() {
		return yoyangName;
	}
	public void setYoyangName(String yoyangName) {
		this.yoyangName = yoyangName;
	}
	
	@Override
	public String toString() {
		return "NuroRES1001U00ReseredDataListItemInfo [receptionTime="
				+ receptionTime + ", patientCode=" + patientCode
				+ ", patientName1=" + patientName1 + ", patientName2="
				+ patientName2 + ", examStatus=" + examStatus + ", updDate="
				+ updDate + ", pkout1001=" + pkout1001 + ", comingDate="
				+ comingDate + ", departmentCode=" + departmentCode
				+ ", receptionNo=" + receptionNo + ", type=" + type
				+ ", doctorCode=" + doctorCode + ", resType=" + resType
				+ ", resUserName=" + resUserName + ", resInputType="
				+ resInputType + ", comingStatus=" + comingStatus + ", newRow="
				+ newRow + ", examState=" + examState + ", examIraiState="
				+ examIraiState + ", resUser=" + resUser + ", ipwonStatus="
				+ ipwonStatus + ", reserComments=" + reserComments
				+ ", reserType=" + reserType 
				+ ", yoyangName=" + yoyangName + "]";
	}
}
